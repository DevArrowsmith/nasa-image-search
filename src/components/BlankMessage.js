import React from 'react';
import styled from 'styled-components';

const BlankStyles = styled.h2`
margin: 20px 20px;
color: #D6D4D0;
`

const BlankMessage = () => {
  return (
    <div data-testid="blank-message">
      <BlankStyles>
        Please enter a search term.
      </BlankStyles>
    </div>
  )
};

export default BlankMessage;