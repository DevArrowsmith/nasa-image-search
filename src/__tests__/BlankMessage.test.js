import React from 'react';
import { render} from '@testing-library/react';
import BlankMessage from '../components/BlankMessage';

test('component BlankMessage renders', () => {
  const { asFragment } = render(<BlankMessage/>);
  const component = asFragment();
  expect(component).toMatchSnapshot();
});
