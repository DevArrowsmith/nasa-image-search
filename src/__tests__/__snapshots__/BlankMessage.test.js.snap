// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component BlankMessage renders 1`] = `
<DocumentFragment>
  <div
    data-testid="blank-message"
  >
    <h2
      class="sc-AxjAm cmwgYd"
    >
      Please enter a search term.
    </h2>
  </div>
</DocumentFragment>
`;
